src/views/NewStudyView.js

```diff
@@ -0,0 +1,97 @@
+import React, {useState} from 'react';
+import {graphql, compose} from 'react-apollo';
+import NewStudyForm from '../forms/NewStudyForm';
+import {Segment, Container, Header, Loader, Icon} from 'semantic-ui-react';
+import {CREATE_STUDY} from '../state/mutations';
+import {MY_PROFILE} from '../state/queries';
+/**
+ * The NewStudyView displays a form to collect details about a new study.
+ */
+
+const NewStudyView = ({match, history, location, createStudy, user}) => {
+  const [newStudyError, setNewStudyError] = useState();
+  const [studyCreating, setStudyCreating] = useState(false);
+  const isAdmin = !user.loading
+    ? user.myProfile.roles.includes('ADMIN')
+    : false;
+  const submitValue = values => {
+    setStudyCreating(true);
+    createStudy({variables: {input: values}})
+      .then(resp => {
+        const studyId = resp.data.createStudy.study.kfId;
+        setStudyCreating(false);
+        history.push(`/study/${studyId}/documents`);
+      })
+      .catch(err => {
+        setNewStudyError(err.message);
+        setStudyCreating(false);
+      });
+  };
+
+  return (
+    <Container as={Segment} vertical basic>
+      {isAdmin ? (
```

Comment:
If you use an AdminRoute for this view, you won't have to have this big conditional render in here.